### ISSUE TITLE
Decreased apolipoprotein levels

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5987]
**Preferred term label:** Decreased apolipoprotein levels

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of apolipoprotein in the blood is below the normal range., PMID: 6099394

**Parent term (use hpo.jax.org/app)** Abnormal apolipoprotein level HP:0025201

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
This could be the parent term for: Decreased apolipoprotein AI level HP:0031799

**Comment by pnrobinson** [comment]
added
